### DISCUSSION TITLE
Screen reader accessibility

### DISCUSSION THREAD

**Discussion by codeofdusk** [DustinBrett/daedalOS#51] [Feature Requests]
I'm totally blind and use [NVDA](https://nvaccess.org).

* Sometimes, I lose focus when opening things (just see the menus and an unlabelled image).
* I can't read the terminal at all. For bonus points, mark it as an [ARIA live region](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) for automatic readout.
    * On that note, strategic ARIA usage could make the experience feel a lot more desktop-like.

**Comment by DustinBrett** [comment]
Hi @codeofdusk,

Thank you very much for this feedback!

I am sorry my website is not as accessible as it could be, that is for sure. Windows 10 afaik has reasonably good accessibility and I did intend to also replicate that, but it had been pushed down in priorities.

I am assembling a new list as I've been getting lots of comments/suggestions since publicizing my site, and accessibility will be given higher priority this year.

**Comment by codeofdusk** [comment]
Thanks for letting me know. I'd be happy to test any changes you make in this area!

**Comment by codeofdusk** [comment]
Why has this been converted?

**Comment by DustinBrett** [comment]
I am just organizing the repo into Issues being bugs and everything else categorized into the discussion area where it makes sense for it to be. I do plan to get to accessibility, but it's not buggy, it just doesn't exist. We can discuss it here and even open a new category for specific accessibility asks. All issues that were not considered bugs were converted.

**Comment by DustinBrett** [comment]
I've done some work on this but I'd love an audit by someone who can help me beef up the a11y support.

**Comment by codeofdusk** [comment]
I'll play with this soon!
